superset/viz.py

```diff
@@ -2849,7 +2849,7 @@ def levels_for(
         for i in range(0, len(groups) + 1):
             agg_df = df.groupby(groups[:i]) if i else df
             levels[i] = (
-                agg_df.mean()
+                agg_df.mean(numeric_only=True)
```

Comment:
new default in 2.0 is `False` https://pandas.pydata.org/docs/whatsnew/v2.0.0.html#deprecations